src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/InputComponent.tsx

```diff
@@ -1,20 +1,77 @@
 import * as React from 'react';
 import '../../styles/shared.css';
 import classNames from 'classnames';
+import NumberFormat from 'react-number-format';
+import { Input } from '@material-ui/core';
 
-export interface IInputProps {
+export interface IInputBaseProps {
   id: string;
   readOnly: boolean;
   required: boolean;
-  formData: any;
+  formatting?: any;
```

Comment:
Would love a defined interface for this prop if possible!

Reply:
@lorang92 Yeah, two minds, same thought :-P https://github.com/Altinn/altinn-studio/issues/5972#issuecomment-844712498

Reply:
Yup, that's a good point. Looking into how we can use the exported interface from `react-number-format` here so that we don't have to maintain the list of available props for the number formatting component ourselves. 

@altinnadmin will respond to your comment, but the short answer is that we would have to maintain our own version of the list of available props in a JSON schema, which might not be the best solution as we would have to maintain it in parallel with the react-number-format library 😞 One suggestion that I tried is to use a link in the description to the current list of available props: https://github.com/Altinn/altinn-cdn/pull/39/files